### ISSUE TITLE
DeprecationWarning with Python 3.7

### ISSUE THREAD

**Issue by dmajda** [dyve/django-bootstrap3#455]
When using django-bootstrap3 with Python 3.7 with warnings enabled, the following deprecation warning is emitted:

```
/usr/local/lib/python3.7/dist-packages/bootstrap3/utils.py:5: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Mapping
```

It would be good if django-bootstrap3 was warning-free on Python 3.7. (In our specific case, this warning pollutes our test suite output.)

### Steps to reproduce

```console
$ python3.7 --version
Python 3.7.1
$ python3.7 -m pip install django django-bootstrap
[...]
$ python3.7 -Wall -c 'import bootstrap3.utils'
/usr/local/lib/python3.7/dist-packages/bootstrap3/utils.py:5: DeprecationWarning: Using or importing the ABCs from 'collections' instead of from 'collections.abc' is deprecated, and in 3.8 it will stop working
  from collections import Mapping
```

**Comment by dyve** [comment]
Thanks! This will be fixed in the next release.

**Comment by dmajda** [comment]
The issue is still present in the newest released version (11.1.0).

**Comment by dmajda** [comment]
The issue is fixed in the newest released version (12.0.3). I was most likely fixed by [this commit](https://github.com/dyve/django-bootstrap3/commit/124b2b038e7dbeb149d02cc5d7a2b2b71b0e74ea), meaning it was already fixed in 12.0.0.
